Examples/HackerNewsClient/HackerNewsClient/Actions/GoBack.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  GoBack.swift
+//  HackerNewsClient
+//
+//  Created by Francesco Di Lorenzo on 15/11/2016.
```

Comment:
Can you update all the headers and conform to the one we use in katana (the one with the MIT license basically)